cmd/srcd/cmd/prune.go

```diff
@@ -16,7 +16,11 @@ var pruneCmd = &cobra.Command{
 			return humanizef(err, "could not prune components")
 		}
 
-		return humanizef(daemon.CleanUp(), "could not clean up")
+		if err := daemon.CleanUp(); err != nil {
```

Comment:
I would better add `if` inside `humanizef` function. It will make client code cleaner and we will avoid similar errors in other places.

Reply:
I have pushed that `if` in ddecbcb.

I don't think we should change the caller style in this PR, it looks like all the rest of the code does
```go
err := call()
if err != nil {
  return humanizef(err, "msg")
}
```
instead of 
```go
return humanizef(call(), "msg")
```

If you think we should change it I'd rather move that discussion to slack or a new issue. And then change all the calls in the same PR.

Reply:
I'm okay with both snippets. The important thing: using any of them doesn't panic anymore :)